qt_ui/windows/QLiberationWindow.py

```diff
@@ -294,6 +294,12 @@ def setGame(self, game: Optional[Game]):
                 self.info_panel.setGame(game)
             self.game_model.set(self.game)
             self.liberation_map.set_game(game)
+
+            if "debriefings" not in vars(game):
+                logging.debug("Adding debriefings attribute to old save game")
+                setattr(
+                    game, "debriefings", [None for i in range(self.game.turn)]
+                )  # adding a debriefing attr to old games
```

Comment:
Just let save compat break. There will always be some unavoidable break per major release. It's not worth the headache of needing these to be optional.

Reply:
man, that's brutal, but ight

Reply:
so since it's already worked out, should it stay or be removed?